dao/src/test/java/com/epam/esm/hibernate/impl/OrderRepositoryImplTest.java

```diff
@@ -0,0 +1,111 @@
+package com.epam.esm.hibernate.impl;
+
+import com.epam.esm.config.ConfigDB;
+import com.epam.esm.exception.DaoException;
+import com.epam.esm.hibernate.OrderRepository;
+import com.epam.esm.persistence.OrderEntity;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.support.AnnotationConfigContextLoader;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please remove * from imports
You can set your Intelij Idea to not add such imports